RadarSDK/Radar.m

```diff
@@ -663,4 +687,15 @@ + (NSDictionary *)dictionaryForLocation:(CLLocation *)location {
     return dict;
 }
 
+- (void)applicationWillEnterForeground {
+    BOOL updated = [RadarSettings updateSessionId];
+    if (updated) {
+        [[RadarAPIClient sharedInstance] getConfig];
+    }
+}
+
+- (void)dealloc {
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
I guess there's an edge case where radar would not have added itself as an observer if `initializeWithPublishableKey` is never called.  I'm assuming `removeObserver` can handle that case.

Reply:
`Radar` is a singleton and unless you call a method (initialize or something else) it will never be initialized and `dealloc` will never be called. I'll squint at this to make sure, though